src/lib/services-manager/dialogs/services_manager.rb

```diff
@@ -243,6 +242,18 @@ def ok_button_help
         help + "<br />"
       end
 
+      # @return [String]
+      def help_note
+        # TRANSLATORS: name of the buttons to save the changes
+        button_text = show_apply_button? ? _("'the Apply' or the 'OK'") : _("the 'OK'")
+
+        format(
+          # TRANSLATORS: note about dialog behavior
+          _("Note: the services will not be modified until all changes are saved by using %{button} button."),
```

Comment:
I am pretty sure that this will break in some languages. It is very likely impossible for the translators to get this right. Avoid to piece together sentences like this; this is bound to fail in some cases.

The original version was easy and safe to translate; this one is not. Even though this one might be more 100% technically correct, it raises problems for the translators. Please revert this to the previous commit and add just the missing "the" articles.